client/views/forgotPassword/forgotPassword.html

```diff
@@ -0,0 +1,20 @@
+<template name="ForgotPassword">
+	<div id="darker">
+	</div>
+	<div class="formcontainer">
+		{{> close_button}}
+		<h1 class="formtitle">Forgot Password</h1>
+		<div class="modaltopmessage">
+			Enter your registered Email
```

Comment:
@singhsanket143 `Enter your registered Email` > `Enter your registered email address`

Reply:
@peter-hank I have updated the text here